samples/sample-04/build.gradle

```diff
@@ -0,0 +1,87 @@
+buildscript {
+    ext {
+        kotlinVersion = "1.3.10"
+        springBootVersion = "2.1.1.RELEASE"
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-allopen:${kotlinVersion}")
+        classpath("org.jetbrains.kotlin:kotlin-noarg:${kotlinVersion}")
+        classpath("io.spring.gradle:dependency-management-plugin:v1.0.6.RELEASE")
+    }
+}
+
+apply plugin: "kotlin"
+apply plugin: "kotlin-spring"
+apply plugin: "org.springframework.boot"
+apply plugin: "io.spring.dependency-management"
+apply plugin: "kotlin-allopen"
+apply plugin: "kotlin-noarg"
+apply plugin: "kotlin-jpa"
+
+
+group "com.example"
+version "0.0.1-SNAPSHOT"
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven { url 'http://repo.spring.io/milestone' }
+}
+noArg{
+    annotation("com.remusrd.notesample.domain.annotation.NoArg")
+}
+allOpen{
+    annotation("com.remusrd.notesample.domain.annotation.Open")
```

Comment:
I feel like we can just remove these two declarations.
First of all - wrong package.
Secondly if we are here, then project even works without these two annotations.

I'm confused, so everybody else is going to be to. 🤷‍♂️